src/backend/looped/world/groundsnow.cpp

```diff
@@ -0,0 +1,29 @@
+#include "backend/looped_command.hpp"
+#include "fiber_pool.hpp"
+#include "gta_util.hpp"
+#include "script.hpp"
+#include "services/tunables/tunables_service.hpp"
+
+namespace big
+{
+	class ground_snow : looped_command
+	{
+		using looped_command::looped_command;
```

Comment:
A bool command may be better suited for this since you're not making use of the on_tick function.

Reply:
I'll move it to on_tick and keep it as a looped_command, explanation in gir489returns's comment.

Reply:
Also is it possible to send you a message on telegram or discord? I have some plans I'd like to present for future PR's and I'm not sure whether they fit here or not.

Reply:
Thanks!